python/pyspark/mllib/feature.py

```diff
@@ -84,11 +84,14 @@ class Normalizer(VectorTransformer):
     >>> nor2 = Normalizer(float("inf"))
     >>> nor2.transform(v)
     DenseVector([0.0, 0.5, 1.0])
+
+    .. versionadded:: 1.2.0
```

Comment:
For `1.10` or `1.1.1`, we still need to use string. This change is not necessary, just one minor comment.